pkg/registry/falcon_registry/registry.go

```diff
@@ -117,6 +121,57 @@ func guessLastTag(tags []string, filter func(string) bool) (string, error) {
 func listDockerTags(ctx context.Context, sys *types.SystemContext, imgRef types.ImageReference) ([]string, error) {
 	tags, err := docker.GetRepositoryTags(ctx, sys, imgRef)
 	if err != nil {
```

Comment:
Should we check for a specific `err` and then execute the `http` request?
Or is that handled indirectly by checking `len(tags)` below.
IIUC, partial list of tags populated in `tags` also is considered an error.

Reply:
Looking at the error now for the string `registry returned invalid tag`